samples/RXPTest/src/Tests/ViewBasicTest.tsx

```diff
@@ -201,7 +201,7 @@ class BasicView extends RX.Component<RX.CommonProps, RX.Stateless> {
 // blockPointerEvents: boolean = false; // iOS and Android only
 // ignorePointerEvents: boolean = false; // web only
 // restrictFocusWithin: boolean = false; // web only
-// limitFocusWithin: boolean = false; // web only
+// limitFocusWithin: LimitFocusType = LimitFocusType.Unlimited; // web only
```

Comment:
Is this really "web only"? Or does it apply to all keyboard-based systems? If it's the latter, delete the "// web only" comment.